database/localdb/localdb.go

```diff
@@ -0,0 +1,65 @@
+package localdb
+
+import (
+	"sync"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/tecbot/gorocksdb"
+
+	"chain/errors"
+)
+
+// DB provides access to a kv store.
```

Comment:
That seems like it must be true, but I'm not sure how to verify that. http://godoc.org/github.com/tecbot/gorocksdb doesn't say anything explicitly. (And checking both the gorocksdb godoc and some of the RocksDB docs for concurrency-related notes seems to mostly yield references to concurrent log compaction.)